components/deck/slide-item.webc

```diff
@@ -112,6 +112,37 @@
       --action: LinkColor;
       --active: ActiveText;
     }
+
+    browser-support::part(link),
```

Comment:
Maybe we could put these in the `support-slide` component?

Reply:
Good call. Moved in https://github.com/oddbird/eleventy-plugin-slide-deck/pull/2/commits/4a0ec36fdf4920381f19aeab8682372394b61954